docs/configuration_options.md

```diff
@@ -368,3 +368,6 @@
 | **<a href="#user-content-mgmt_agent_backup_config" name="mgmt_agent_backup_config">mgmt_agent_backup_config</a>**  | Manager agent backup general configuration: checkers, transfers, low_level_retries | N/A | SCT_MGMT_AGENT_BACKUP_CONFIG
 | **<a href="#user-content-mgmt_reuse_backup_snapshot_name" name="mgmt_reuse_backup_snapshot_name">mgmt_reuse_backup_snapshot_name</a>**  | Name of backup snapshot to use in Manager restore benchmark test | N/A | SCT_MGMT_REUSE_BACKUP_SNAPSHOT_NAME
 | **<a href="#user-content-mgmt_skip_post_restore_stress_read" name="mgmt_skip_post_restore_stress_read">mgmt_skip_post_restore_stress_read</a>**  | Skip post-restore c-s verification read in the Manager restore benchmark tests | N/A | SCT_MGMT_SKIP_POST_RESTORE_STRESS_READ
+| **<a href="#user-content-auto_scaling_action_type" name="auto_scaling_action_type">auto_scaling_action_type</a>**  | Refers to autoscale task like scaleout, scalein etc that needs to be performed at 90% storage| N/A | SCT_AUTO_SCALING_ACTION_TYPE
```

Comment:
I would not call it auto-scaling, just scaling. Auto-scaling is a different feature that we would like to implement in the future - automatic change of provisioned capacity units when we are close to the utilization limit.